app/views/pages/home.html.haml

```diff
@@ -80,10 +91,17 @@
                                                value: "Submit"}
 .rackspace
   .row
-    %p
-      <span class="logo-h">h</span>ackEDU is generously hosted by
-      = link_to 'Rackspace', 'https://www.rackspace.com/'
-      %span.right
+    .small-12.large-4.columns
+      %p
+        We can be reached anytime at #{mail_to 'team@hackedu.us'} or 
+        at <a href="https://webchat.freenode.net/?channels=hackedu">#hackEDU</a>
+        on freenode
+    .small-12.large-4.columns
+      %p
+        <span class="logo-h">h</span>ackEDU is generously hosted by
+        = link_to 'Rackspace', 'https://www.rackspace.com/'
```

Comment:
Is this still necessary now that we have Rackspace listed as a sponsor on the homepage?